l10n/en/firefox/browsers/best-browser.ftl

```diff
@@ -0,0 +1,51 @@
+best-browser-this-is-one-of-the = This is one of the many reasons to choose the { -brand-name-firefox } browser: { -brand-name-firefox } blocks third-party trackers by default. We have other reasons and we’ll get into those later.
+best-browser-a-browser-that-puts = A browser that puts safety first.
+best-browser-remember-the-last-massive = Remember the last massive data breach? If not, it’s probably because it happens so often. Companies hold on to customer data, like their personal or financial information, and hackers steal it. If you’re making safety a priority, then a secure internet browser is the best browser for you.
+best-browser-there-are-a-few-ways = There are a few ways a browser can help its users stay secure. A browser that is up to date with the latest security tech can help protect your computer and websites from unwanted visitors, such as malware or computer viruses.
+
+# Variables:
+# $data (url) - link to https://www.mozilla.org/privacy/firefox/
+# $privacy (url) - link to https://www.mozilla.org/privacy/firefox/
+best-browser-the-second-is-not-storing = The second is not storing too much user data. Hackers can’t steal what’s not there, which is why { -brand-name-firefox } keeps a minimum amount of information about its users. <a href="{ $data }">{ -brand-name-firefox } knows</a> if you use the browser and your general location <a href="{ $privacy }">but not the name of your childhood pet or your favorite color.</a>
+
+best-browser-last-but-not-least = Last but not least, a safe browser should offer tools to help you keep an eye on your accounts. Think of alerts that go straight to your email if any of your accounts get breached or icons that tell you whether a website is encrypted, (i.e., if it’s a good idea to enter your credit number on a shopping site).
+
+# Variables:
+# $monitor (url) - link to https://monitor.firefox.com
+best-browser-firefox-is-offering = { -brand-name-firefox } is offering something new to keep you safe: <a href="{ $monitor }">{ -brand-name-firefox-monitor }</a>. It’s a free service that will alert you if there are any public hacks on your accounts and let you know if your accounts got hacked in the past. Another neat feature is the Green Lock. It looks like a small green icon at the top left side of the browser window. If you’re on { -brand-name-firefox } and see the green lock, it means the website is encrypted and secure. If the lock is grey, you might want to think twice about entering any sensitive information.
```

Comment:
> Another neat feature is the Green Lock.

I'm unsure why "Green Lock" is capitalized like this. Is it a proper name?

Reply:
You're probably right here, but I'm going to try and avoid rotating on copy that's already been reviewed and well translated. If we come to update the page in the future then let's fix it 👍